source/whats-new.txt

```diff
@@ -46,9 +50,30 @@ What's New in 4.11
 .. note:: Upgraded Netty Version
 
    The {+driver-short+} now tests with version 4.1.87.Final of the
-   ``io.netty:netty-all`` package. This change affects you only if you use the 
+   ``io.netty:netty-all`` package. This change affects you only if you use the
    ``SslContext`` class that Netty provides.
 
+New features of the 4.11 driver release include:
+
+- The ``getStats()`` and ``isCapped()`` instance methods of the
```

Comment:
It is odd to present this as a new feature.  Let's consolidate all this information in the Upcoming Breaking Changes section, so that it's easy to find for future upgraders.